src/parser/priest/discipline/constants.js

```diff
@@ -11,22 +11,22 @@ export const ABILITIES_AFFECTED_BY_HEALING_INCREASES = [
   SPELLS.LEECH.id,
 ];
 
-export const DAMAGING_SPELLS_THAT_CAUSE_ATONEMENT = [
-  SPELLS.MAGIC_MELEE.id, // Shadow Fiend Melee
-  SPELLS.LIGHTSPAWN_MELEE.id, // Lightspawn Melee
-  SPELLS.SMITE.id,
-  SPELLS.PENANCE.id,
-  SPELLS.HALO_DAMAGE.id,
-  SPELLS.SHADOW_WORD_PAIN.id,
-  SPELLS.PURGE_THE_WICKED_TALENT.id,
-  SPELLS.PURGE_THE_WICKED_BUFF.id,
-  SPELLS.POWER_WORD_SOLACE_TALENT.id,
-  SPELLS.SCHISM_TALENT.id,
-  SPELLS.DIVINE_STAR_DAMAGE.id,
-  SPELLS.HOLY_NOVA.id,
-];
+export const ATONEMENT_DAMAGE_SOURCES = {
```

Comment:
Changed this to a hashmap for a minor performance bump, this was being looped over multiple times for each damaging event that came in for Discipline.